### DISCUSSION TITLE
support get column by string

### DISCUSSION THREAD

**Discussion by joebnb** [SeaQL/sea-orm#2214] [Q&A] [answered]
would sea orm support get column by string,this case often happen in table sort or filt something,current if i want implement sort func i should get :1. is ascend or descend 2.which column will be affect. i need write every query builder as : 

```rs
fn get_column_type_by_name(column_name: &str) -> Option<Type> {
    let entity = Model::Entity;
    let column_type = match column_name {
        "id" => Some(Column::Id),
        "name" => Some(Column::Name),
        "email" => Some(Column::Email),
        _ => None,
    };
    column_type
}

fn main() {
    if let Some(column_type) = get_column_type_by_name("name") {
        println!("get column is: {:?}", column_type);
    }
}
```

if there have some function to drop in string and return a col type would be help full

expect
```rs
fn main() {
    if let Some(column_type) = entity:Column:get_column_by_name("name") { // direct return Some Column
        println!("get column is: {:?}", column_type);
    }
}
```

**Comment by tyt2y3** [comment, marked as answer]
Here it is!
https://github.com/SeaQL/sea-orm/blob/aa626fc1fddcaec4351d52eebd30b8d412b1c87e/src/entity/column.rs#L513-L537

**Reply by joebnb** [reply]
thanks for reply
